grade-scripts/README.md

```diff
@@ -0,0 +1,72 @@
+Privacy grade scripts
+====
+
+These are a set of scripts that run the DDG privacy grade algorithm against a set of sites in bulk, so we can reason about it more easily.
+
+Prerequisites
+---
+
+Tested on NodeJS 8.11.x.
+
+A few datasets aren't publicly available just yet: `https_list.txt`, `https_autoupgrade_list.txt`, original `polisis.json`.
```

Comment:
Also worth noting https_list.txt and https_autoupgrade_list are required, and whoever is testing needs to mkdir data/symlinked and move the files in there